### PR TITLE
Add sentry logging

### PR DESCRIPTION
**Pull request by ml8**
## Goal

Log exceptions to sentry

## Todos:
- [x] Create sentry project for kevel proxy ([done](https://mozilla.sentry.io/projects/kevel-proxy-gcp/?project=4506543461302272))
- [x] Configure logging for proxy (this pr)
- [x] Update prod/staging configs (pr to accompany this one)
- [ ] Connect sentry project to pagerduty

### CODE REVIEW

File: app/provider/sentry_provider.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    dsn = conf.sentry['dsn']
+    traces_sample_rate = conf.sentry['traces_sample_rate']
+    profiles_sample_rate = conf.sentry['profiles_sample_rate']
+    if not dsn:
+        return
+    sentry_sdk.init(
+            dsn=dsn,
+            traces_sample_rate=traces_sample_rate,
+            profiles_sample_rate=profiles_sample_rate,
+            environment=conf.env)
```
Comment (cbguder) [inline app/provider/sentry_provider.py:17]: Super nit

```suggestion
            environment=conf.env,
    )
```

**Review by cbguder — APPROVED** [review]

**Review by bkochendorfer — DISMISSED** [review]
lgtm, re @cbguder nit I wonder if there is a linter/formatter you all could agree on to get that automagically

**Comment by cbguder** [comment]
> lgtm, re @cbguder nit I wonder if there is a linter/formatter you all could agree on to get that automagically

Yeah, we could use [Black](https://black.readthedocs.io/en/stable/) but this codebase is not long for this world so it's probably not worth setting it up (which I guess also puts my nit into perspective 🙃).

**Comment by ml8** [comment]
> lgtm, re @cbguder nit I wonder if there is a linter/formatter you all could agree on to get that automagically

Yes we definitely need to align on a style guide and associated linters! Does Mozilla have a py style guide? It looks like firefox has this: https://firefox-source-docs.mozilla.org/code-quality/coding-style/coding_style_python.html but I can't find a corresponding guide for services. I suppose we should adopt firefox standards but let's parking lot this for tomorrow's meeting

**Review by cbguder — APPROVED** [review]

**Review by dmueller — APPROVED** [review]

**Pull request merged**